lstchain/scripts/lstchain_data_r0_to_dl1.py

```diff
@@ -151,21 +152,35 @@ def main():
         except Exception as e:
             log.error(f'Configuration file could not be read: {e}')
             sys.exit(1)
+    else:
+        config = standard_config
+
+    # Add to configuration config the parameters provided through command-line,
+    # which supersede those in the file:
+    config['source_config']['EventSource']['max_events'] = args.max_events
+
+    lst_event_source = config['source_config']['LSTEventSource']
+    lst_event_source['EventTimeCalculator']['ucts_t0_dragon'] = \
+        args.ucts_t0_dragon
+    lst_event_source['EventTimeCalculator']['dragon_counter0'] = \
+        args.dragon_counter0
+    lst_event_source['EventTimeCalculator']['ucts_t0_tib'] = \
+        args.ucts_t0_tib
+    lst_event_source['EventTimeCalculator']['tib_counter0'] = \
+        args.tib_counter0
+    lst_event_source['PointingSource']['drive_report_path'] = \
+        args.pointing_file
+    lst_r0_corrections = lst_event_source['LSTR0Corrections']
+    lst_r0_corrections['drs4_pedestal_path'] = args.pedestal_file
```

Comment:
I am maybe missing something, but what if we do not give any `pedestal_file` through the command line?

Reply:
Behaviour would be the same as before, i.e. it is a required argument. But this means those settings cannot be configured through the json file as of now.

Reply:
> Behaviour would be the same as before, i.e. it is a required argument. But this means those settings cannot be configured through the json file as of now. Will fix.

Done